run_project_tests.py

```diff
@@ -695,12 +699,20 @@ def _run_tests(all_tests, log_name_base, failfast, extra_args):
             testname = '%.3d %s' % (int(testnum), testbase)
             should_fail = False
             suite_args = []
+            final_extra = extra_args.copy()
             if name.startswith('failing'):
                 should_fail = name.split('failing-')[1]
             if name.startswith('warning'):
                 suite_args = ['--fatal-meson-warnings']
                 should_fail = name.split('warning-')[1]
-            result = executor.submit(run_test, skipped, t.as_posix(), extra_args + suite_args,
+            if (name in {'cmake', 'common', 'platform-linux'} and
+                system_compiler in {'gcc', 'clang'} and
+                    testname not in {'099 threads', '177 initial c_args'}):
```

Comment:
This should be `99 threads` without the leading zero, no?

Reply:
https://github.com/mesonbuild/meson/blob/f6ad6a577a014f3a9c84ec727f5b5dac84d5570a/run_project_tests.py#L699 makes is %03d leading zero 3-digit integer

Reply:
`t.name` has the original name. Use that instead to avoid this confusing extra zero.

Reply:
I used t.name now, thanks.